common/src/main/java/com/wynntils/utils/colors/CommonColors.java

```diff
@@ -28,4 +25,12 @@ public final class CommonColors {
     public static final CustomColor MAGENTA = CustomColor.fromInt(0xff0083);
     public static final CustomColor ORANGE = CustomColor.fromInt(0xff9000);
     public static final CustomColor WHITE = CustomColor.fromInt(0xffffff);
+
+    // Wynncraft's custom effects done via resourcepack shaders
+    public static final CustomColor RAINBOW = CustomColor.fromInt(0x00f000); // Animated rainbow
+    public static final CustomColor GRADIENT =
+            CustomColor.fromInt(0x00f004); // Animated gradient from #f56217 to #0b486b
+    public static final CustomColor FADE = CustomColor.fromInt(0x00f008); // Smooth animated fade from #5af082 to black
+    public static final CustomColor BLINK =
+            CustomColor.fromInt(0x00f00c); // Animated blinking between #e63232 and black
```

Comment:
I really don't prefer inline comments breaking lines because of spotless, this should be fine now. 

```suggestion
    // region Wynncraft's custom effects done via resourcepack shaders
    
     // Animated rainbow
    public static final CustomColor RAINBOW = CustomColor.fromInt(0x00f000);
     // Animated gradient from #f56217 to #0b486b
    public static final CustomColor GRADIENT = CustomColor.fromInt(0x00f004);
     // Smooth animated fade from #5af082 to black
    public static final CustomColor FADE = CustomColor.fromInt(0x00f008);
    // Animated blinking between #e63232 and black
    public static final CustomColor BLINK = CustomColor.fromInt(0x00f00c); 
    
    // endregion
```

Reply:
I guess you did think the same thing as me :)

Reply:
Okay, this is fine as is.